deployment/ansible/roles/ckan-odp-configuration/defaults/main.yml

```diff
@@ -7,4 +7,5 @@ ckan_smtp_starttls: "False"
 ckan_smtp_password: ""
 ckan_database_backups: "/root/db_backups"
 ckan_database_backup_script: "/root/backup_db.sh"
+ckan_licenses: "/usr/local/share/licenses.json"
```

Comment:
How does this get pulled into CKAN -- does CKAN automatically look for licenses in this location? If so it might be good to add a comment about that.